.github/workflows/review-checks.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ["34", "35", "rawhide"]
+        version: ["34", "35", "36", "rawhide"]
```

Comment:
I know EOL for 34 is 6/7 but since I don't anticipate any feature releases between that time, I'd like to propose we remove version 34 now.

Reply:
I mean I would rather keep it. If we will do maintenance for any reason it is good to have all versions and make sure we are not breaking something. We can remove 34 anytime..